docs/server/source/production-deployment-template/node-on-kubernetes.rst

```diff
@@ -488,6 +488,27 @@ You can check its status using: ``kubectl get pvc -w``
 Initially, the status of persistent volume claims might be "Pending"
 but it should become "Bound" fairly quickly.
 
+.. Note::
+    Defualt Reclaim Policy for dynamically created persistent volumes is ``Delete``
```

Comment:
"The default Reclaim Policy..." (i.e. Add 'The' at the beginning of the sentence and change the spelling of 'Defualt`)